src/SmoFacade/Database.cs

```diff
@@ -15,10 +15,35 @@ public class Database
     private readonly Microsoft.SqlServer.Management.Smo.Database _database;
     private readonly Server _server;
 
+    public Database(DatabaseConfig databaseConfig)
+    { 
+        string connectionString = databaseConfig.ConnectionString;
+        string dbName = databaseConfig.DatabaseName;
+
+        var sqlConnection = new System.Data.SqlClient.SqlConnection(connectionString);
+        var sqlServerConnection = new Microsoft.SqlServer.Management.Common.ServerConnection(sqlConnection);
+        var sqlSmoServer = new Microsoft.SqlServer.Management.Smo.Server(sqlServerConnection);
+        _database = new Microsoft.SqlServer.Management.Smo.Database(sqlSmoServer, dbName); 

```

Comment:
put the following `using` statements at the top of this file so that these names aren't so verbose
```C#
  using System.Data.SqlClient;
  using Microsoft.SqlServer.Management.Common;
  using Microsoft.SqlServer.Management.Smo;
```

Reply:
Not needed anymore